helpers-debian/copyright

```diff
@@ -1,11 +1,30 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: libtrexio
-Source: <url://github.com/TREX-CoE/trexio>
-#
-# Please double check copyright with the licensecheck(1) command.
+Upstream-Contact: Evgeny <posenitskiy@irsamc.ups-tlse.fr>
+Source: <https://github.com/TREX-CoE/trexio>
+
+Files: debian/*
+Copyright: 2022 Evgeny <posenitskiy@irsamc.ups-tlse.fr>
+License: GPL-2+
```

Comment:
Careful! We have BSD 3-clause license, not GPL

Reply:
The TREXIO source code and related files have BSD 3-Clause, you can scroll down.

Files in the `debian/` folder are generated by the Debian-related tool and should probably go under the corresponding license.

NB: not all TREX codes/repos have the BSD 3-Clause license (e.g. `qmckl_bench`).